{"name": "test_timeout_error", "status": "passed", "description": "Test timeout error produces standard format.", "attachments": [{"name": "log", "source": "47efd9e9-e865-4a11-9bda-cc1ee4cb4216-attachment.txt", "type": "text/plain"}], "start": 1757656801696, "stop": 1757656801697, "uuid": "768d59f8-cd90-4d3a-9ca2-6c7bd3fd7694", "historyId": "d47899ef8add7312ed513ce369486002", "testCaseId": "d47899ef8add7312ed513ce369486002", "fullName": "tests.unit.test_standardized_llm_errors.TestStandardizedErrors#test_timeout_error", "labels": [{"name": "tag", "value": "asyncio"}, {"name": "parentSuite", "value": "tests.unit"}, {"name": "suite", "value": "test_standardized_llm_errors"}, {"name": "subSuite", "value": "TestStandardizedErrors"}, {"name": "host", "value": "Joshuas-MacBook-Pro-2.local"}, {"name": "thread", "value": "36177-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.unit.test_standardized_llm_errors"}], "titlePath": ["tests", "unit", "test_standardized_llm_errors.py", "TestStandardizedErrors"]}